app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -12,6 +12,6 @@ class VerifyCsrfToken extends BaseVerifier
      * @var array
      */
     protected $except = [
-        //
+        'api/*',
```

Comment:
You can exclude this middleware by putting the API endpoints in the [`api` middleware group](https://github.com/DoSomething/rogue/blob/db087e225a3dbddf2e66f0bf318d1d079229fcbd/app/Http/Kernel.php#L34-L36), rather than [`web`](https://github.com/DoSomething/rogue/blob/db087e225a3dbddf2e66f0bf318d1d079229fcbd/app/Http/Kernel.php#L26-L32).